components/docs-chef-io/content/automate/restore.md

```diff
@@ -52,6 +52,13 @@ Ensure access for the backup type used:
          # """
     ```
 
+1. To restore a backup to a machine with less memory than the original system, adjust for the appropriate lower memory settings by adding the following to the configuration file:
```

Comment:
@mattray @yzl I incorporated this content into the Restore Prerequisites section as y'all previously discussed. Does this information need to follow the "create a `patch.toml` file instructions on line 33 rather than a direct change to the `config.toml` file? a la
> create a `patch.toml` file that specifies the lower memory amount and provide it at restore time:

Reply:
@mjingle Thanks for doing this. What I would suggest is rewriting the item that starts "To restore a backup to a host with a different fully-qualified domain name..." to something like "Patch the configuration at restore time if the restore host has a different FQDN and/or less memory than the host on which the backup was taken," and then provide the config for FQDN and the config for memory as sub-items.

Reply:
No worries, @yzl. So in my edits, I want to keep each piece of information about FQDN or about lower memory in its separate numbered point for clarity. Also, I'm trying to keep the beginning sentence structure for line 55 the same as the other numbered sentences in this section, so folks can compare the sentences and figure out what applies to their situation in a quicker way. 

What I want to confirm is whether adding the following -

>  [elasticsearch.v1.sys.runtime]
>      heapsize = 4096m

- should be done 1) by editing the configuration file OR 2) by creating and applying a `patch.toml` file (like it is for FQDN).

Reply:
@mjingle They should do 2.

Reply:
@yzl That's what I suspected! Thank you for confirming. I'll be sure to update accordingly then.